docs/system-models.rst

```diff
@@ -53,16 +54,123 @@ necessary to specify a particular storage systems.
+and the name of the attribute of the submodel separated by periods.
+For example, the resistance at fully charge is "resistance.full".
+
+Controlling Which Parameters Are Updatable
+++++++++++++++++++++++++++++++++++++++++++
+
+No parameters of the ``HealthVariable`` are treated as updatable by default.
+As a result, no estimation scheme will alter their values.
+
+Mark a variable as updatable by marking the submodel(s) holding that variable as updatable and
+the name of the variable to the :attr:`updatable` of its submodel.
+Marking "resistance.empty" is achieved by
+
+.. code-block:: python
+
+    model.updatable.add('resistance')
+    model.resistance.updatable.add('empty')
+
+or using the :meth:`mark_updatable` utility method
+
+.. code-block:: python
+
+    model.mark_updatable('resistance.empty')
+
+All submodels along the path to a specific parameter must be updatable for it to be updatable.
+For example, "resistance.full" would not be considered updatable if the "resistance" submodel is not updatable
+
+.. code-block:: python
+
+    model.updatable.remove('resistance')
+    model.resistance.mark_updatable('full')  # Has no effect yet because 'resistance' is fixed
+
+Setting Values of Parameters
+++++++++++++++++++++++++++++
+
+Provide a list of new values and a list of names to the ``update_parameters`` function.
+
+.. code-block:: python
+
+    model.updatable.add('resistance')  # Allows resistance fields to be updated
+    model.update_parameters([[0.1]], ['resistance.full'])
+
+or omit the specific names to set all updatable variables
+
+.. code-block:: python
+
+    assert model.updatable_names == ['resistance.full', 'resistance.empty']
+    model.update_parameters([[0.2, 0.1]])  # As a (1, 2) array for 1-sized batch of 2 values
+
+Defining the Cell Physics
+-------------------------
+
+All storage systems are represented using a :class:`~moirae.models.base.CellModel`
+that provides two functions:
+
+1. updating transient state given inputs, and
+2. predicting outputs given state and inputs.
+
+Cell models hold no state themselves and only implement the physics
+that describes how the state of a battery system should evolve with time.
+Attributes of a cell model adjust the how the calculations are performed
+or are resource-specific configuration,
+such as a path to external components.
+
+Changes in the ASOH for a cell are described as :class:`~moirae.models.base.DegradationModel`.
+Such models provide a function which updates the current state of health provided
+new inputs, transient state, and measurements.
+
+.. ::
+
+    We still need to....
 
-All storage systems are represented using a ``CellModel``.
-The model class defines how the transient states change given
-an inputs applied to a system defined using certain health parameters.
+    1. Describe where any parameters for the degredation model come from
+    2. Indicate if there are any additional states held by the degradation model
```

Comment:
The short answer is yes. But, which states, will be dependent on the degradation model we are implementing...

Reply:
Agreed. FYI, these don't comments are just intended for us later and don't render in the HTML version